include/swift/Basic/Timer.h

```diff
@@ -33,7 +33,8 @@ namespace swift {
   public:
     explicit SharedTimer(StringRef name) {
       if (CompilationTimersEnabled == State::Enabled)
-        Timer.emplace(name, StringRef("Swift compilation"));
+        Timer.emplace(name, StringRef("Swift compilation"), StringRef("swift"),
```

Comment:
For all I know the extra StringRef is completely unnecessary here and was just maintained because it was already there before...